raiden/network/transport/matrix/transport.py

```diff
@@ -238,6 +243,10 @@ def message_is_in_queue(data: _RetryQueue._MessageData) -> bool:
                     message=msg_data.message,
                     reason="Message was removed from queue",
                 )
+            else:
+                if next(msg_data.expiration_generator):
+                    # Only (re-)send this message if the retry period has elapsed
```

Comment:
could you please add `and the message has not been delivered` or something in this form here?